xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-ui/src/main/resources/XWiki/Mentions/MentionsTranslations.fr.xml

```diff
@@ -51,5 +51,6 @@ XWiki.Mentions.MentionsConfigurationClass_mentionsColor=Couleur des mentions
 XWiki.Mentions.MentionsConfigurationClass_mentionsColor.hint=Couleur des mentions affichées à l'utilisateur connecté (note : utilisée comme valeur de l'attribut css background-color).
 XWiki.Mentions.MentionsConfigurationClass_selfMentionsColor=Couleurs des mentions de soi-même
 XWiki.Mentions.MentionsConfigurationClass_quoteActivated=Activer les citations
-XWiki.Mentions.MentionsConfigurationClass_quoteActivated.hint=Définit si une citation du texte entourant une mention est incluse dans les notifications de mention d'utilisateur.</content>
+XWiki.Mentions.MentionsConfigurationClass_quoteActivated.hint=Définit si une citation du texte entourant une mention est incluse dans les notifications de mention d'utilisateur.
+mentions.tip=Lors de la rédaction dans l'éditeur WYSIWYG, saisissez @ puis des lettres et vous obtiendrez des suggestions pour mentionner un utilisateur.</content>
```

Comment:
bof :) I would rather ay "Lors de l'édition en WYSIWYG, ..."

Reply:
Or "Lorsque vous éditez en WYSIWYG".

Reply:
You can now edit it on l10n.xwiki.org.

Reply:
> You can now edit it on l10n.xwiki.org.

Done, thx